From e7c15b97b518b96134e3ecf9b7848376dc057eed Mon Sep 17 00:00:00 2001
From: Mohamed Sentissi <msentissi@anaconda.com>
Date: Wed, 16 Oct 2024 17:06:51 -0400
Subject: [PATCH] remove framework classifiers

---
 pyproject.toml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index db581d9..96588d9 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -14,8 +14,6 @@ authors = [
 ]
 classifiers = [
   "Development Status :: 5 - Production/Stable",
-  "Framework :: OpenTelemetry",
-  "Framework :: OpenTelemetry :: Exporters",
   "Intended Audience :: Developers",
   "License :: OSI Approved :: Apache Software License",
   "Programming Language :: Python",
-- 
2.45.2

